src/strucwght.m

```diff
@@ -0,0 +1,16 @@
+function W = strucwght(alldata)
+%estimates structure weight before glueing
+
+a = alldata{1,1};
+
+specwght = a(:,6);
+vol = a(:,7);
+W = 0;
+
+for i = 1:length(specwght)
+    
+    W = W + (vol(i)/(12^3))*specwght(i);
+    
+end
+
+end
```

Comment:
For all above, just add comments so that people who don't know about this function can understand what it does. Good Job, Just missing some comments! Thank you for your contribution.